### PR TITLE
Configuring GUI

### PR DESCRIPTION
**Pull request by vsakharevych**

### CODE REVIEW

File: src/filemanager/filemanager.cpp [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
     if (file.open(QIODevice::ReadOnly))
     {
         QTextStream stream(&file);
-        QString readResult = stream.readAll();
+        QString rReadResult = stream.readAll();
         file.close();
-        return readResult;
+        return rReadResult;
     }
     throw QException();
```
Comment (PeterLysyk) [inline src/filemanager/filemanager.cpp]: maybe better idea is to create your own exception

File: src/mainwindow/mainwindow.h [hunk 2]
```diff
@@ -35,9 +36,8 @@ class MainWindow : public QMainWindow
     BottomPanelDock *mpBottomPanelDock;
     MDIArea *mpDocsArea;
     QString mCurrentUserName;
-
     void setupMainMenu();
-    void saveDocument(CodeEditor* pDoc, QString fileName);
+    void saveDocument(CodeEditor *pDoc, const QString &fileName);
     void openDoc(QString fileName);
     bool checkIfOpened(const QString& fileName)const;
```
Comment (PeterLysyk) [inline src/mainwindow/mainwindow.h]: 1)bool checkIfOpened(const QString &fileName)const;
2) Maybe better name would be "isOpenned"

File: src/chatwindow/chatwidget.cpp [hunk 3]
```diff
@@ -49,3 +54,41 @@ void ChatWidget::setUsersList(const QStringList &usersList)
...
+    QString insertion = "<" + userName + "> " + message;
+    feed += insertion;
+    mpFeed->setPlainText(feed);
+}
+
+void ChatWidget::onUserToConnectSelected(QListWidgetItem *item)
+{
+    QString userName = item->text();
+    int position = userName.lastIndexOf(QChar{':'});
+    emit userToConnectSelected(userName.mid(position + 2));
```
Comment (PeterLysyk) [inline src/chatwindow/chatwidget.cpp]: why (position + 2)?

File: src/startpage/startpage.cpp [hunk 4]
```diff
@@ -11,54 +11,65 @@ StartPage::StartPage(QWidget *parent) :
...
+
+    //lblFont.setBold(true);
+    QPalette palette;
+    palette.setColor(QPalette::WindowText, Qt::blue);
 
     // creating & configuring new file button
-    pNewBtn = new QPushButton(tr("   New File"));
-    setupButton(pNewBtn, style()->standardIcon(QStyle::SP_FileIcon), maxButtonsWidth);
-    connect(pNewBtn, &QPushButton::pressed, this, &StartPage::newBtnPressed);
+    mpNewBtn = new QPushButton(tr("   New File"));
```
Comment (PeterLysyk) [inline src/startpage/startpage.cpp]: For what is this space between " and New?

**Review by PeterLysyk — APPROVED** [review]

File: src/filemanager/filemanager.cpp (hunk 1 above)
Comment (vsakharevych) [reply src/filemanager/filemanager.cpp]: done

File: src/mainwindow/mainwindow.h (hunk 2 above)
Comment (vsakharevych) [reply src/mainwindow/mainwindow.h]: done

File: src/chatwindow/chatwidget.cpp (hunk 3 above)
Comment (vsakharevych) [reply src/chatwindow/chatwidget.cpp]: We process here the following string:
"Connect to: userName"
I cannot begin search from the end as user name might consist of several words or any combination of characters

File: src/startpage/startpage.cpp (hunk 4 above)
Comment (vsakharevych) [reply src/startpage/startpage.cpp]: Tried to adjust spacing between icon & text on button.
Fixed that: now icon is aligned to the right & text - to the centre

File: src/newfilewizard/newfilewizard.cpp [hunk 5]
```diff
@@ -139,17 +157,21 @@ void NewFileDialog::onCreateFile()
...
+    } catch (const FileOpeningFailure&)
     {
         QMessageBox::warning(this, "Error", "Unable to create file");
         return;
     }
     QMessageBox::information(this, "File Created", "Specified file has been successfully created.");
 
     // check if filename is an empty string
-    if(fileName.isEmpty())
+    if (fileName.isEmpty())
```
Comment (FedorenkoAndriy) [inline src/newfilewizard/newfilewizard.cpp]: mFileName  = fileName.isEmpty() ? QString() : dirName + '/' + fileName;

File: src/newfilewizard/newfilewizard.cpp [hunk 6]
```diff
@@ -110,24 +122,30 @@ void NewFileDialog::onCreateFile()
     QString dirName = mpDirLbl->text();
 
     // check if directory is not empty
-    if(dirName.isEmpty())
+    if (dirName.isEmpty())
     {
         QMessageBox::warning(this, "Wrong directory",
```
Comment (FedorenkoAndriy) [inline src/newfilewizard/newfilewizard.cpp]: It is better to create constants for the hardcoded text

**Review by FedorenkoAndriy — APPROVED** [review]

File: src/newfilewizard/newfilewizard.cpp (hunk 5 above)
Comment (vsakharevych) [reply src/newfilewizard/newfilewizard.cpp]: fixed that

File: src/newfilewizard/newfilewizard.cpp (hunk 6 above)
Comment (vsakharevych) [reply src/newfilewizard/newfilewizard.cpp]: Fixed that

**Pull request merged**